products/sle/main.pm

```diff
@@ -677,10 +699,14 @@ if (is_jeos) {
     loadtest "console/suseconnect_scc";
 }
 
+
```

Comment:
That's not a related change. If you want to change the structure for a good reason please make it at least its own commit with a good reasoning